docs/guides/universal-profile/transfer-lyx.md

```diff
@@ -8,32 +8,61 @@ import TabItem from '@theme/TabItem';
+
+- In our [**Edit a Universal Profile**](./edit-profile.md) guide, we saw the basics of interacting with our UP to edit our profile info. We did so by interacting via the KM.
+
+We previously saw how to use `setData(...)` to update data in our UP contract's storage. Let's now look at `execute(...)`.
 
-Therefore to interact with our UP, we will need to encode the function call of our UP and pass the **payload** to the [`execute(payload)`](../../standards/smart-contracts/lsp6-key-manager.md#execute) function on the KM.
+### Basics of the `execute(...)` function
 
-Transferring LYX from a UP is as simple as making a standard [`CALL`](../../standards/universal-profile/lsp6-key-manager.md#permission-values) to any `address`, attaching some **value** to the call. You can transfer LYX from a UP via the [`execute(...)`](../../standards/smart-contracts/lsp0-erc725-account.md) function in the UP contract.
+The [`execute(operation,to,value,data)`](../../standards/smart-contracts/erc725-contract.md#execute---erc725x) function from [ERC725X](../../standards/lsp-background/erc725.md#erc725x---generic-executor) enable us to use our UP to interact with other addresses, such as transferring LYX or calling other smart contracts. This function takes four arguments (see [ERC725 API docs](../../standards/smart-contracts/erc725-contract.md#execute---erc725x)).
 
-The parameters of the function will be as follow:
 
+We can use this function to transfer LYX from our UP to any address (including another UP). Transferring LYX from our UP is as simple as making a standard [`CALL`](../../standards/universal-profile/lsp6-key-manager.md#permission-values) to any `address`, attaching some **value** to the call.
+
+For a regular LYX transfer, the parameters will be:
 - `_operation`: `0` (for `CALL`).
 - `_to`: the `address` we want to send LYX to (Externally Owned Account or contract address).
 - `_value`: the amount of LYX we want to transfer (in Wei).
-- `_data`: empty payload (`0x`).
+- `_data`: empty (`0x` since we are just transferring LYX).
+
+### Interacting via the Key Manager
+
+Most of the functions on the UP contract, such as [`setData(...)`](../../standards/smart-contracts/erc725-contract.md#setdata---erc725y) and [`execute(...)`](../../standards/smart-contracts/erc725-contract.md#execute---erc725x) can only be called by the [`owner`](../../standards/smart-contracts/erc725-contract.md#owner). Therefore if we want to use our UP to do meaningful things, **all interactions should go through the KM**.
+
+![](/img/guides/transfer-lyx-interaction-via-key-manager.jpeg)
 
-Since we are just making a simple LYX transfer, the fourth parameter `_data` will be empty.
+In order to transfer LYX from our UP, we need to perform the following steps:
+
+1. abi-encode the [`execute(operation,to,value,data)`](../../standards/smart-contracts/erc725-contract.md#execute---erc725x) function call of our UP.
```

Comment:
@CallumGrindle @Hugoo I would like to have a convention for ABI. As it is an abbreviation and set term. I would suggest to always use uppercase. One example: `NFT` is always used in uppercase, even if the sentence would allow `non-fungible token` in lowercase.

Reply:
Yep I agree uppercase ABI is good